tools/simulator/libsimulator/lib/ProjectConfig/ProjectConfig.cpp

```diff
@@ -421,18 +409,11 @@ void ProjectConfig::parseCommandLine(const vector<string> &args)
             vector<string> pathes = split((*it), ';');
```

Comment:
I guess that delimiter character is ':' better than ';'.
';' is same as command delimiter for unix-shell.